### PR TITLE
Implement global site

### PR DESCRIPTION
**Pull request by smarzban**

### CODE REVIEW

File: molo/globalsite/templates/globalsite/country_site.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+
+<h1 class="">{% trans "Choose your country:" %}</h1>
+<div class="">
+  {% for region in regions %}
+    {{region}}
+    {% for country in region.country_sites.all %}
+      <li class="">
+        <a href="{% url 'molo.globalsite:set_country' country.code %}" class="">
+          <div class="">{% if country.flag %}
```
Comment (miltontony) [inline molo/globalsite/templates/globalsite/country_site.html]: I think the if statement needs to be outside the `<div>` and the `class=""` is not necessary.

File: molo/globalsite/templatetags/globalsite_tags.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+from django import template
+from wagtail.wagtailcore.models import Site
+
+register = template.Library()
+
+
+@register.simple_tag(takes_context=True)
+def current_country(context):
```
Comment (miltontony) [inline molo/globalsite/templatetags/globalsite_tags.py]: This doesn't appear to be used. Also, you should have access to `request.site` by default.

**Review by miltontony — COMMENTED** [review]
Please enable coverage

**Comment by smarzban** [comment]
@miltontony coverage enabled 👌

**Comment by smarzban** [comment]
@miltontony ready for re-review

**Comment by miltontony** [comment]
👍 🍰 🌟

**Pull request merged**